internal/cephfs/nodeserver.go

```diff
@@ -151,7 +151,10 @@ func (*NodeServer) mount(ctx context.Context, volOptions *volumeOptions, req *cs
 	klog.V(4).Infof(util.Log(ctx, "cephfs: mounting volume %s with %s"), volID, m.name())
 
 	if err = m.mount(ctx, stagingTargetPath, cr, volOptions); err != nil {
-		klog.Errorf(util.Log(ctx, "failed to mount volume %s: %v"), volID, err)
+		klog.Errorf(util.Log(ctx,
+			"failed to mount volume %s: %v Check dmesg logs if required."),
```

Comment:
Maybe add a `.` between `%v Check`?

Reply:
Period (.) is added by the value of %v. If I add another it looks odd :)

See this:
Mounting arguments: -t ext4 -o _netdev,debug,invalidOption,defaults /dev/rbd0 /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-199ae7ce-288b-4959-95d5-b28600920200/globalmount/0001-0009-rook-ceph-0000000000000001-3ab79687-9f44-11ea-8391-0242ac110010
Output: mount: /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-199ae7ce-288b-4959-95d5-b28600920200/globalmount/0001-0009-rook-ceph-0000000000000001-3ab79687-9f44-11ea-8391-0242ac110010: wrong fs type, bad option, bad superblock on /dev/rbd0, missing codepage or helper program, or other error**.**
**.** Check dmesg logs if required.

Reply:
Ok, thanks for checking!